installing/installing_azure/installation-config-parameters-azure.adoc

```diff
@@ -0,0 +1,12 @@
+:_content-type: ASSEMBLY
+[id="installation-config-parameters-azure"]
+= Installation configuration parameters for Azure
+include::_attributes/common-attributes.adoc[]
+:context: installation-config-parameters-azure
+:platform: Azure
+
+toc::[]
+
+Before you deploy an {product-title} cluster on Microsoft Azure, you provide parameters to customize your cluster and the platform that hosts it. When you create the `install-config.yaml` file, you provide values for the required parameters through the command line. You can then modify the `install-config.yaml` file to customize your cluster further.
+
```

Comment:
**Peer review**

The goal of this PR is to refactor existing content. I acknowledge that this existing content needs some work. Once I have completed the refactor for all providers, I plan on rewriting this abstract and pushing it through QE review. Please see previous reviews:

- [Alibaba](https://github.com/openshift/openshift-docs/pull/64049#discussion_r1310262310)
- [IBM Power VS](https://github.com/openshift/openshift-docs/pull/64103#discussion_r1311755915)
- [GCP](https://github.com/openshift/openshift-docs/pull/61722#discussion_r1247021585)